recipe/build.sh

```diff
@@ -10,3 +9,16 @@ bash configure  --prefix=$PREFIX $OPTS
 make
 make check
 make install
+
+if [ "$(uname)" == "Darwin" ]; then
+    # Based on
+    # https://github.com/Homebrew/homebrew-core/blob/master/Formula/emacs.rb
+    mv nextstep/Emacs.app $PREFIX/Emacs.app
```

Comment:
This seems very strange. It installs both the emacs.app and the stuff from `make install` above (from `--disable-ns-self-contained`). It seems like this would install duplicate stuff. But as far as I can tell, it's what the Homebrew recipe does.